src/Sulu/Bundle/ContactBundle/Api/Account.php

```diff
@@ -1088,12 +1103,24 @@ public function removeMedia(MediaInterface $media)
     /**
      * Get medias.
      *
+     * @deprecated Please use getMedia() instead.
```

Comment:
We usually mention the full method name and the version since when it is deprecated. So this should be something like this:

```
@deprecated since 2.0, use Account::getMedia instead
```